src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -122,13 +146,8 @@ class MediaBaseViewer extends BaseViewer {
         // Try catch is needed due to weird behavior when src is removed
         try {
             if (this.mediaEl) {
-                this.mediaEl.removeEventListener('timeupdate', this.setTimeCode);
-                this.mediaEl.removeEventListener('progress', this.progressHandler);
-                this.mediaEl.removeEventListener('volumechange', this.updateVolumeIcon);
-                this.mediaEl.removeEventListener('playing', this.playingHandler);
-                this.mediaEl.removeEventListener('pause', this.pauseHandler);
-                this.mediaEl.removeEventListener('ended', this.resetPlayIcon);
-                this.mediaEl.removeEventListener('seeked', this.seekHandler);
+                this.removeEventListenersForMediaElement();
```

Comment:
What's the reasoning for moving these out? It doesn't seem to be used outside of `destroy`.

Reply:
Mainly just to consolidate, I'll add `loaddata` and `error` as well